avocado/plugins/list.py

```diff
@@ -53,7 +53,6 @@ def _get_test_suite(self, paths):
 
     def _get_test_matrix(self, test_suite):
         test_matrix = []
-
```

Comment:
Why removing only this line? File contains other blank lines in similar context...